Makefile

```diff
@@ -0,0 +1,29 @@
+ifdef MVN_FLAGS
```

Comment:
For my education; why use a make file?

Reply:
Well mainly since it is an easy interface for

1. chaining tasks like `make build test` and define task dependencies like `build: build-parent build-core` without installing any frameworks, tooling or anything
2. Using some tool of this sort lets me bundle all build task in one place and reused them for devs and ci, which reduces frictions, keeps it DRY and reproducable

mvn itself is kind of hard to use for those orchestrations and using bash would lose point 1. (still give 2.)